### ISSUE TITLE
Indic Transliteration Issue

### ISSUE THREAD

**Issue by kmadathil** [kmadathil/sanskrit_parser#141]
Reported as https://github.com/sanskrit-coders/indic_transliteration/issues/39

If you see something like this (Devanagari output where SLP1 is expected):
```
> sanskrit_parser vakya astyuttarasyAMdiSi --input SLP1                                                    Interpreting input loosely (strict_io set to false)
INFO     Input String: astyuttarasyAMdiSi
INFO     Input String in SLP1: अस्त्युत्तरस्यान्दिशि
```

Please downgrade indic_transliteration with `pip install indic_transliteration==1.9.4`

**Comment by avinashvarna** [comment]
Should we update our dependencies in setup.py to use indic_transliteration>=1.9.7 (now that it's fixed)?

**Comment by kmadathil** [comment]
Yes we should.

**Comment by kmadathil** [comment]
I have updated install_requires to skip 1.9.5 and 1.9.6. 
The version with the fix 1.9.7 is not on pypi yet
